### ISSUE TITLE
vagrant halt is broken

### ISSUE THREAD

**Issue by coldnebo** [rails/rails-dev-box#115]
I'm seeing this on an Ubuntu 14.04 host:

```
lkyrala@ziggy:../rails-dev-box$ vagrant halt
==> default: Attempting graceful shutdown of VM...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

shutdown -h now

Stdout from the command:



Stderr from the command:

stdin: is not a tty
/var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:163:in `rescue in root': Could not locate Gemfile or .bundle/ directory (Bundler::GemfileNotFound)
	from /var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:159:in `root'
	from /var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:71:in `bundle_path'
	from /var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:419:in `configure_gem_home'
	from /var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:407:in `configure_gem_home_and_path'
	from /var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:57:in `configure'
	from /var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:118:in `definition'
	from /var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:88:in `setup'
	from /var/lib/gems/2.3.0/gems/bundler-1.11.2/lib/bundler.rb:99:in `require'
	from /var/lib/gems/2.3.0/gems/sucker_punch-2.0.0/bin/shutdown:4:in `<top (required)>'
	from /usr/local/bin/shutdown:23:in `load'
	from /usr/local/bin/shutdown:23:in `<main>'
```

I'm a complete newb to this vm, so I'm not sure why the wrapping on these functions.  

Is there something I need to do here to install correctly?  I was able to setup the vm ok and it appears to run rails tests just fine, but it doesn't shut down correctly.

`vagrant halt --force` works.

Thanks!

**Comment by fxn** [comment]
This repository provides a virtual machine for Rails core development, but questions or isses which are specific to Vagrant would be better solved in Vagrant channels. You could maybe try [here](http://stackoverflow.com/questions/tagged/vagrant) for example.

**Comment by coldnebo** [comment]
@fxn this isn't a vagrant issue, it's all rails-dev-box.

Why is 'shutdown' in the rails vm being wrapped with a ruby/bundler script that can fail in a root context from within the vm?

**Comment by fxn** [comment]
You are totally right. It turns out I could not reproduce because I had run bundle install without the `:job`. group in the host machine and apparently that file in `/usr/local/bin` [is installed by sucker_punch](https://github.com/brandonhilkert/sucker_punch/blob/master/bin/shutdown).

That's a key system command, how weird it is hijacked... let me investigate.

**Comment by fxn** [comment]
[Opened an issue in sucker_punch](https://github.com/brandonhilkert/sucker_punch/issues/166). Closing here because does not seem something specific to the VM.

**Comment by coldnebo** [comment]
Thanks!

**Comment by coldnebo** [comment]
brandonhilkert/sucker_punch#166 closed with a bump to 2.0.1.  

Sounds like the easiest way to revert (clean the bins) and get the new gem is to simply `vagrant destroy` and `vagrant up`.  It doesn't seem worth it to add versioning (`'~> 2.0.1'`) to [the rails Gemfile](https://github.com/rails/rails/blob/fbab5c7e6a9df3533f8b0fa8d6609ac9ec3ba399/Gemfile#L51)

I just wanted to document the final steps in case other people have this issue.

Thanks!

**Comment by fxn** [comment]
Awesome @coldnebo, thanks for the followup :).
